src/sys/socket/consts.rs

```diff
@@ -278,15 +288,10 @@ mod os {
             MSG_OOB,
             MSG_PEEK,
             MSG_DONTWAIT,
-            #[cfg(not(target_os = "dragonfly"))]
             MSG_CTRUNC,
-            #[cfg(not(target_os = "dragonfly"))]
             MSG_TRUNC,
-            #[cfg(not(target_os = "dragonfly"))]
             MSG_EOR,
-            #[cfg(any(target_os = "linux", target_os = "android"))]
```

Comment:
Damn, was reading as a blacklist there, and seeing the symbols defined for them I removed it. Should definitely use the same form for all `#[cfg]` ^^